apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore/Logging/LoggerOptions.cs

```diff
@@ -12,40 +12,50 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+using Google.Api;
 using Google.Api.Gax;
 using Google.Cloud.Diagnostics.Common;
 using Microsoft.Extensions.Logging;
 
 namespace Google.Cloud.Diagnostics.AspNetCore
 {
-    // TODO(talarico): Add option for logging to other resources and organizations.
-
     /// <summary>
     /// Options for a <see cref="GoogleLogger"/>.
     /// </summary>
     public sealed class LoggerOptions
     {
+        /// <summary>The global resource.</summary>
+        internal static readonly MonitoredResource s_globalResource = new MonitoredResource { Type = "global" };
+
         /// <summary>The minimum log level.</summary>
         public LogLevel LogLevel { get; }
 
+        /// <summary>The monitored resource. See: https://cloud.google.com/logging/docs/api/v2/resource-list </summary>
+        public MonitoredResource MonitoredResource { get; }
+
         /// <summary>The buffer options for the logger.</summary>
         public BufferOptions BufferOptions { get; }
 
-        private LoggerOptions(LogLevel logLevel, BufferOptions bufferOptions)
+        private LoggerOptions(LogLevel logLevel, MonitoredResource monitoredResource, BufferOptions bufferOptions)
         {
             LogLevel = GaxPreconditions.CheckEnumValue(logLevel, nameof(logLevel));
+            MonitoredResource = GaxPreconditions.CheckNotNull(monitoredResource, nameof(monitoredResource));
```

Comment:
This null check and the one below are unnecessary since you provide defaults in Create. Again, use Debug.Assert here if you're worried about future changes violating these invariants.